pkgs/development/python-modules/unstructured/default.nix

```diff
@@ -57,39 +113,7 @@
   grpcio,
 }:
 let
-  version = "0.16.8";
-  optional-dependencies = {
-    huggingflace = [
-      langdetect
-      sacremoses
-      sentencepiece
-      torch
-      transformers
-    ];
-    local-inference = [ unstructured-inference ];
```

Comment:
This breaks eval for `unstructured-api`

Reply:
Yes i saw, i basically have to do the same for that package :-(